src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java

```diff
@@ -79,12 +79,15 @@ else if (config.database.maintainInterval < 0)
                             config.deployment.workerService);
                     break;
                 case "pybossa":
-                    platformInstance = new PyBossaPlatform(config.deployment.workerService,
+                    PyBossaPlatform pyBossa = new PyBossaPlatform(config.deployment.workerService,
                             platform.apiKey,
                             platform.url,
                             platform.name,
                             platform.projectId,
                             platform.calibrationsAllowed);
+                    pyBossa.init();
```

Comment:
it initializes the platform and needs a connection to it. I think you should still be able to construct the platform then